client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx

```diff
@@ -154,20 +154,34 @@ class DomainManagementNavigationEnhanced extends React.Component {
 	}
 
 	getTransferDomain() {
-		const { selectedSite, translate, domain } = this.props;
-		const { expired } = domain;
+		const { moment, selectedSite, translate, domain } = this.props;
+		const { expired, isLocked, transferAwayEligibleAt } = domain;
 
 		if ( expired && ! this.isDomainInGracePeriod() ) {
 			return null;
 		}
 
-		// NOTE: remember to add translate to the description string once you start working on it
+		let description;
+
+		if ( transferAwayEligibleAt ) {
+			description = translate( 'Outbound transfers available after %(startDate)s', {
+				args: {
+					startDate: moment( transferAwayEligibleAt ).format( 'LL' ),
+				},
+				comment: '%(startDate)s is a date string, e.g. April 1, 2020',
+			} );
+		} else if ( isLocked ) {
+			description = translate( 'Transfer lock: on' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Transfer lock: off' )` *ES Score: 6*